CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 
 - Fix Docker storage not pulling correct flow path - [#968](https://github.com/PrefectHQ/prefect/pull/968)
 - Set Docker storage to pull master git branch until next version is released - [#968](https://github.com/PrefectHQ/prefect/pull/968)
+- Fix `run_flow` loading to decode properly by use cloudpickle - [#978](https://github.com/PrefectHQ/prefect/pull/978)
```

Comment:
For the record, I'm not sure how verbose we should be in the changelog about these sorts of things, since this new environments interface was changed in this release history - this "Fix" isn't really a fix on top of 0.5.2

Reply:
Hmm good point, we need to come up with an idea of what kind of change really warrants an update of the changelog